src/mixins/dom.js

```diff
@@ -0,0 +1,34 @@
+// DomMixin
+//  ---------
+
+import Backbone from 'backbone';
+
+export default {
+  createBuffer() {
+    return document.createDocumentFragment();
+  },
+
+  appendChildren(el, children) {
+    Backbone.$(el).append(children);
+  },
+
+  beforeEl(el, sibling) {
+    Backbone.$(el).before(sibling);
+  },
+
+  replaceEl(newEl, oldEl) {
+    Backbone.$(oldEl).replaceWith(newEl);
+  },
+
+  detachContents(el) {
+    Backbone.$(el).contents().detach();
+  },
+
+  attachHTML(el, html) {
+    Backbone.$(el).html(html);
+  },
+
+  removeEl(el) {
+    Backbone.$(el).remove();
```

Comment:
Ah it does look pretty trivial.

And I totally get the overriding thing.. but this internal mixin could be just ignoring the argument
```
  removeEl(el) {
    this.$el.remove();
  }
```